src/collections/_documentation/data-management/rollups.md

```diff
@@ -1,21 +1,48 @@
 ---
-title: 'Rollups & Grouping'
+title: 'Grouping & Fingerprints'
 sidebar_order: 0
 ---
 
-An important part of Sentry is how it aggregates similar events together and creates rollups. This turns out to be a pretty complex issue and it can be confusing for users about why some information might not be grouped correctly.
+An important part of Sentry is how similar events are grouped together. This turns out to be a pretty complex issue and it can be confusing for users about why some information might not be grouped correctly.
```

Comment:
Wordsmithing suggestion (pre-existing stuff, but while we're here...):

An important part of Sentry is the way similar events are grouped together. This turns out to be a pretty complex issue, and it's not always obvious why events are being grouped the way they are.